deploy/crds/cincinnati.openshift.io_v1alpha1_cincinnati_cr.yaml

```diff
@@ -7,13 +7,4 @@ spec:
   registry: "quay.io"
   repository: "openshift-release-dev/ocp-release"
   certConfigMapKey: ""
-  # There two ways to specify the source of the Cincinnati graph data.
-  # The first is through a github repository specified by these three
-  # parameters.
-  gitHubOrg: "openshift"
-  gitHubRepo: "cincinnati-graph-data"
-  branch: "master"
-  # The second is through an init container specified by graphDataImage.
-  # When specified, graphDataImage takes precedence over gitHubOrg,
-  # gitHubRepo, and branch.
-  #graphDataImage: "your-registry/your-repo/your-init-container"
\ No newline at end of file
+  graphDataImage: "quay.io/rwsu/cincinnati-graph-data-container:latest"
```

Comment:
Should we publish a default image in https://quay.io/repository/cincinnati ?

Reply:
I hesitate to create a situation where someone might start using an image we produce as a production mechanism for consuming OCP update data. I think we need to depend on OCP to deliver that critical data and not become redistributors of it.

So I think it's better to enable people to easily build their own. If we want to have our own build for test and dev purposes, we can perhaps name it something very obviously non-production quality like `unreliable-demo-graph-data` or something.

Reply:
My comment is with respect to developers.  Denoting that in the name would be a good idea.

Reply:
I'll revert the value back to "your-registry/your-repo/your-init-container". We can change it to something concrete once a non-production image is published somewhere.